chart/values.yaml

```diff
@@ -528,3 +503,111 @@ mysql:
   extraVolumeMounts: |
     - mountPath: /docker-entrypoint-initdb.d
       name: init-scripts
+
+rabbitmq:
+  fullnameOverride: "messagebus"
+  persistence:
+    enabled: false
+  replicaCount: 1
+  auth:
+    username: ""
+    password: ""
+    tls:
+      enabled: true
+      existingSecret: messagebus-certificates-secret-core
+      existingSecretFullChain: true
+  metrics:
+    enabled: true
+    podAnnotations:
+      prometheus.io/scrape: "true"
+      prometheus.io/port: "9500"
```

Comment:
I know we have that on the messagebus helm template, but they have become unnecessary since some months ago. Let's remove it :)

Reply:
fixed